server/app/views/admin/programs/ProgramSettingsEditView.java

```diff
@@ -0,0 +1,109 @@
+package views.admin.programs;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.form;
+import static j2html.TagCreator.h1;
+import static j2html.TagCreator.input;
+import static j2html.TagCreator.p;
+
+import com.google.inject.Inject;
+import j2html.TagCreator;
+import j2html.tags.specialized.ButtonTag;
+import j2html.tags.specialized.DivTag;
+import j2html.tags.specialized.FormTag;
+import play.mvc.Http.HttpVerbs;
+import play.mvc.Http.Request;
+import play.twirl.api.Content;
+import services.program.ProgramDefinition;
+import views.BaseHtmlView;
+import views.admin.AdminLayout;
+import views.admin.AdminLayout.NavPage;
+import views.admin.AdminLayoutFactory;
+import views.style.StyleUtils;
+
+/** Renders a page for editing program-level settings. */
+public final class ProgramSettingsEditView extends BaseHtmlView {
+  private static final String ELIGIBILITY_TOGGLE_ID = "eligibility-toggle";
+  private static final String ELIGIBILITY_IS_GATING_DESCRIPTION =
+      "When using eligibility conditions, you can set the experience to gating or non-gating. The"
```

Comment:
Same as below, but can we check with Miriam about how final this copy is? Something like:


> When enabled, applicants can submit applications even if the eligibility criteria are not met. When disabled, applications must meet all eligibility criteria in order to submit an application.


has slightly more natural language.

Reply:
The copy isn't final since Miriam is still working on the copy doc; meant to mention that in the PR description but forgot, sorry. I like your suggestions though, and switched to using those for now. We can run it by Miriam later and update it if needed.